test/clj_pgp/test/encryption.clj

```diff
@@ -106,6 +108,37 @@
               (is @error-occured? "A PGP error was simulated but not passed to the error handler."))))))))
 
 
+(deftest decrypting-pgp-file-terminated-with-junk
+  (testing "`reduce-messages`"
+    (is (= "🐢"
+           (pgp-msg/reduce-messages
+             (io/input-stream
+               (io/resource "terminated-without-junk-bytes.txt.gpg"))
```

Comment:
Building on the resource comment above, if you renamed the `test/resources` directory to `test/samples`, then this could be referenced as:
```suggestion
               (io/resource "samples/terminated-without-junk-bytes.txt.gpg"))
```

Reply:
e.g. the `test/keys/` directory with sample keyrings.

Reply:
Yes, though I don't like the `resources` name specifically, because it's confusing given the use of the top-level directory of the same name being on the classpath where you _wouldn't_ include the directory in the `io/resource` path argument.